dotcom-rendering/src/lib/unifyPageContent.tsx

```diff
@@ -25,6 +25,9 @@ export const unifyPageContent = ({
 				{!!elementHtml && (
 					<div dangerouslySetInnerHTML={{ __html: elementHtml }} />
 				)}
+				{/* Interactive atoms on standard articles are not meant to access
+				the main page; this is what interactive articles are for. */}
+				<script>window.parent = window;</script>
```

Comment:
Would it be 'safer' to put this up in the <head> instead? Can elementHtml ever contain inline-js for example?